src/reducers/localeSlice.ts

```diff
@@ -0,0 +1,21 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit"
+
+export interface LocaleState {
+  locale: string
```

Comment:
Do you actually plan on storing more values besides this one string?  If not, the wrapping object isn't necessary - the entire slice state could just be the one string.

Reply:
Not necessarily you're right - I changed it to a string.